src/CUDAdrv.jl

```diff
@@ -3,21 +3,24 @@ __precompile__()
 module CUDAdrv
 
 using Compat
-import Compat.String
+using Compat.String
```

Comment:
Using `using Test.func` on individual names *does not*  add the exported names of the `Test` module to be searched  as in `using Test`, but acts like `import Test.func`  but without allowing accidental function extensions. Perhaps this point should be better clarified in the docs.

BTW why is it even allowed to `import Test` a whole module? Perhaps that should be tightened up.

E.g.

```julia

julia> module Test

       export foo,goo

       foo() = 1
       goo() = 2
       hoo() = 3
       end
Test

julia> using Test.hoo

julia> hoo(x) = x
ERROR: error in method definition: function Test.hoo must be explicitly imported to be extended

julia> foo()
ERROR: UndefVarError: foo not defined

julia> goo()
ERROR: UndefVarError: goo not defined

julia> hoo()
3

## restart and try 


julia> import Test.hoo

julia> hoo(x) = x
hoo (generic function with 2 methods)

julia> foo()
ERROR: UndefVarError: foo not defined

julia> goo()
ERROR: UndefVarError: goo not defined

julia> hoo()
3
```